Source/LinkedList.c

```diff
@@ -2,5 +2,48 @@
  * @file
  * @brief Implementation of singly-linked list.
  */
-
+#include <stdio.h>
+#include <stdlib.h>
 #include "LinkedList.h"
+
+
```

Comment:
Nuke one of these newlines so that you never have more than one in a row (your formatter should do this for you)